README.md

```diff
@@ -1,11 +1,15 @@
-# React + Vite
+# TODO-List APP 
+  React v3 + Vite app for Jay class
 
-This template provides a minimal setup to get React working in Vite with HMR and some ESLint rules.
+# Description
 
-Currently, two official plugins are available:
+# Installation instructions
```

Comment:
Consider what these steps are directing. What you want to do is provide exact instructions for someone to follow, which will allow them to run your project on their machine. Not how to create a Vite template.